tests/power_cycle/test_powercycletools.py

```diff
@@ -0,0 +1,96 @@
+from pprint import pprint
+
+import matplotlib.pyplot as plt
+import pytest
+
+from bluemira.base.look_and_feel import bluemira_print
+from bluemira.power_cycle.tools import (
+    _add_dict_entries,
+    _join_delimited_values,
+    adjust_2d_graph_ranges,
+    validate_axes,
+)
+
+# Simple vectors for sample plotting
+sample_x = [1, 2]
+sample_y = [3, 4]
+
+bluemira_print("Test Power Cycle Tools")
+
+
+def test_add_dict_entries(visualize=None):
```

Comment:
`visualize` will never be set when the test is run it will always be none. 

```suggestion
def test_add_dict_entries():
```